src/ResultPrinter.js

```diff
@@ -0,0 +1,49 @@
+import { Console } from "@woowacourse/mission-utils";
+
+class ResultPrinter {
+  static printResults(carNames, roundResults) {
+    const winners = this.findWinners(carNames, roundResults); // 우승자 찾기
+    this.printWinners(winners); // 최종 결과 출력
+    this.printRoundResults(carNames, roundResults); // 각 라운드 결과 출력
```

Comment:
```suggestion
    const winners = this.findWinners(carNames, roundResults); // 우승자 찾기
    this.#printRoundResults(carNames, roundResults); // 각 라운드 결과 출력
    this.#printWinners(winners); // 최종 결과 출력
```
저도 보니깐 `printWinners`와 `printRoundResults`의 순서가 바뀐 것 같습니다.
추가적으로 `printWinners`와 `printRoundResults`메서드도 외부에서 호출되지 않는다면  `#printWinners`와 `#printRoundResults`으로 바꿔주시는게 좋습니다!